contrib/copy-tex/copy-tex.webpack.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * This is the webpack entry point for KaTeX. As ECMAScript doesn't support
+ * CSS modules natively, a separate entry point is used.
+ */
+import './copy-tex.css';
+import './copy-tex.js';
```

Comment:
This change doesn't seem strictly necessary.  Was thine thinking to make it follow the same pattern as the main katex build?